plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraType.java

```diff
@@ -91,81 +95,16 @@
 
 public class CassandraType
 {
-    public enum Kind
-    {
-        BOOLEAN,
-        TINYINT,
-        SMALLINT,
-        INT,
-        BIGINT,
-        FLOAT,
-        DOUBLE,
-        DECIMAL,
-        DATE,
-        TIMESTAMP,
-        ASCII,
-        TEXT,
-        VARCHAR,
-        BLOB,
-        UUID,
-        TIMEUUID,
-        COUNTER,
-        VARINT,
-        INET,
-        CUSTOM,
-        LIST,
-        SET,
-        MAP,
-        TUPLE,
-        UDT,
-    }
-
-    private final Kind kind;
-    private final Type trinoType;
-    private final List<CassandraType> argumentTypes;
-
-    public CassandraType(
-            Kind kind,
-            Type trinoType)
-    {
-        this(kind, trinoType, ImmutableList.of());
-    }
-
-    @JsonCreator
-    public CassandraType(
-            @JsonProperty("kind") Kind kind,
-            @JsonProperty("trinoType") Type trinoType,
-            @JsonProperty("argumentTypes") List<CassandraType> argumentTypes)
-    {
-        this.kind = requireNonNull(kind, "kind is null");
-        this.trinoType = requireNonNull(trinoType, "trinoType is null");
-        this.argumentTypes = ImmutableList.copyOf(requireNonNull(argumentTypes, "argumentTypes is null"));
-    }
-
-    @JsonProperty
-    public Kind getKind()
-    {
-        return kind;
-    }
+    @SuppressWarnings("unused")
+    private final TypeManager typeManager;
```

Comment:
Please remove `TypeManager` from the 1st commit (I added this to my example just for showcase).